generated/thrift/rpc.thrift

```diff
@@ -104,8 +111,13 @@ struct HealthResult {
 	2: required string status
 }
 
+struct IDWithNamespace {
+    1: required string ns
+    2: required string id
+}
+
 struct WriteRequest {
-	1: required string id
+    1: required IDWithNamespace idWithNamespace
```

Comment:
We could, but just an FYI that means `WriteBatchRequest` is no longer simply a list of `WriteRequest` because we end up with something like
```
struct NamedDatapoint {
    1: required string id
    2: required Datapoint datapoint
}

struct WriteRequest {
    1: required string nameSpace
    2: required NamedDatapoint namedDatapoint
}

struct WriteBatchRequest {
    1: required string nameSpace
	2: required list<NamedDatapoint> elements
}
```
so the pooling logic for `WriteRequest` and `WriteBatchRequest` in host_queue will need to change too.